Assets/Scripts/Ability.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class Ability
+{
+    private GameObject player;
+    private Animator animator;
+
+    public string name;
+    public float coolDown;
+    public float damageToEnemy;
+    private string triggerName;
+
+    public float timeOfLastUse;
+
+    public bool unlocked =  true;
+    
+
+    public Ability(string name, GameObject player, float damageToEnemy, float coolDown, string triggerName)
+    {
+        this.name = name;
+        this.player = player;
+        this.damageToEnemy = damageToEnemy;
+        this.coolDown = coolDown;
```

Comment:
triggerName isn't set here, leads to animations not really happening

Reply:
Oh I was wondering why that wasn't happening

Reply:
Fixed